chia/full_node/full_node_api.py

```diff
@@ -835,10 +835,7 @@ async def declare_proof_of_space(
             # 3. In a future sub-slot that we already know of
 
             # Grab best transactions from Mempool for given tip target
-            aggregate_signature: G2Element = G2Element()
-            block_generator: Optional[BlockGenerator] = None
-            additions: Optional[list[Coin]] = []
-            removals: Optional[list[Coin]] = []
+            new_block_gen: Optional[NewBlockGenerator]
```

Comment:
I think this should default to None

Reply:
note that this doesn't define the variable, it just declares its type, it doesn't exist at this point yet